assets/js/jquery-tiptip/jquery.tipTip.js

```diff
@@ -76,16 +76,16 @@
 						});
 					}
 				} else if(opts.activation == "focus"){
-					org_elem.focus(function(){
+					org_elem.on( 'focus', function(){
 						active_tiptip();
-					}).blur(function(){
+					}).on( 'blur', function(){
 						deactive_tiptip();
 					});
 				} else if(opts.activation == "click"){
-					org_elem.click(function(){
+					org_elem.on( 'click', function(){
 						active_tiptip();
 						return false;
-					}).hover(function(){},function(){
+					}).on( 'hover', function(){},function(){
```

Comment:
Just wanted to note that this might not be the best replacement for the hover function, see: https://github.com/jquery/jquery-migrate/blob/master/warnings.md#jqmigrate-jqueryfnhover-is-deprecated
There are also three more occurrences in this file where hover is called.

Reply:
Thanks @mikkamp !